RecoLocalCalo/HGCalRecProducers/plugins/HGCalRecHitWorkerSimple.cc

```diff
@@ -59,7 +59,9 @@ HGCalRecHitWorkerSimple::HGCalRecHitWorkerSimple(const edm::ParameterSet& ps) :
     rcorr_.push_back(1.0 / corr);
   }
   // here for scintillator
-  rcorrscint_ = ps.getParameter<double>("sciThicknessCorrection");
+  const auto& rcorrscint = ps.getParameter<double>("sciThicknessCorrection");
+  rcorrscint_ = 1.0 / rcorrscint;
```

Comment:
this could just be `rcorrscint_ = 1.0 / ps.getParameter<double>("sciThicknessCorrection");`